features/workspace/products/product/label-tags/toolbar/ToolbarMarkerGroup.tsx

```
@@ -39,17 +45,38 @@ const ToolbarMarkerGroup = ({
+        size="icon-sm"
         variant={variant === "ghost" ? "default" : "outline"}
         className="rounded-r-none border-r-0 cursor-pointer"
         onClick={() => handleMarkerSelection(currentMarkerType)}
       >
         <currentMarkerType.icon />
-      </Toggle>
-      <Popover open={popoverOpen} onOpenChange={setPopoverOpen}>
+      </Toggle> */}
+      <ButtonGroup>
+        {markers.markerTypes.map((markerType) => (
+          <Tooltip>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)